src/server/prismicStore.js

```diff
@@ -7,6 +7,7 @@ import logger from './logger';
 const cache = {
   callsToAction: [],
   startHere: {},
+  todos: [],
```

Comment:
Ah - I was wondering about that.  Is f9f93798 on the right track?  I got rid of the todos reference here and also split out the reducer into separate functions for managing `todos` / `callsToAction` / `startHere` (the last two being trivial noops right now) 

...but then got a bit lost trying to implement the "Prismic data should be added via a redux action" part, since I wasn't sure how to do that without making `prismicStore` a client-side dependency too or else having separate reducers for client and server, which I'm guessing is Not Supposed To Happen.  So I suspect I'm just missing something obvious?